servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilder.java

```diff
@@ -105,9 +105,18 @@ public StreamingHttpClient buildStreaming() {
                             defaultReqRespFactory(buildContext.httpConfig().asReadOnly(),
                                     executionContext.bufferAllocator())));
 
-            // Need to wrap the top level client (group) in order for non-relative redirects to work
-            urlClient = maxRedirects <= 0 ? urlClient :
-                    new RedirectingHttpRequesterFilter(false, maxRedirects).create(urlClient);
+            if (maxRedirects > 0 || followRedirectsInitializer != null) {
+                final RedirectingHttpRequesterFilter.Builder redirectBuilder =
+                        new RedirectingHttpRequesterFilter.Builder().allowNonRelativeRedirects(true);
```

Comment:
We pre-set the default value for users because we know that this client is able to follow non-relative redirects. This is to keep pre-existing behavior and to be consistent with other 3-party clients (like AHC). If users want to disable non-relative redirects, they can override this default via configuration.

Reply:
I'd love that to be added as a comment to this line.

Reply:
After the recent change https://github.com/apple/servicetalk/pull/1792/commits/cc2a1116c1abc0827568d2bac6e72fa2e1d7a444 users of multi-address client have to always provide their own `RedirectConfig`. So, we don't need tp pre-set any defaults for them.